### DISCUSSION TITLE
"Google blocked the request"

### DISCUSSION THREAD

**Discussion by maddiemii** [TransformerOptimus/SuperAGI#296] [General]
For some reason I'm getting the following:
uperagi-celery-1           | [2023-06-10 07:09:52,612: WARNING/ForkPoolWorker-8] Tool: GoogleSearch
superagi-celery-1           | [2023-06-10 07:09:53,751: WARNING/ForkPoolWorker-8] Error: 404
superagi-celery-1           | [2023-06-10 07:09:53,752: WARNING/ForkPoolWorker-8] Google blocked the request. Trying again...

- I do have a Custom Search API set up, and a key generated. The key has unrestricted access.
- I also have the key set up for programmable search

Anyone else struggled with this? I'm a bit confused.

**Comment by EthanHuntJr** [comment]
Im not having the exact issue as you however I am getting the following issue with the google search tool even though I have a Customer Search API key and a Programmable Search Key.

P.S Ignore the search 😂its just to show it doesnt work

Tool GoogleSearch returned: Error1: 'content', KeyError, args: {'query': 'Lewis Hamilton girlfriend'}

**Reply by maddiemii** [reply]
Thank you for the response! My issue though is that my agent isn't getting any results back from searching. So it has no way of searching the internet :(

**Comment by neelayan7** [comment]
Can you try generating new keys and try again?
